src/fxcorrelator_bengops.py

```diff
@@ -1355,13 +1363,13 @@ def spead_issue_all(corr, beams=[], from_fpga=True):
     :param from_fpga:
     :return:
     """
-    spead_static_meta_issue(corr, beams=beams)
+    spead_static_meta_issue(corr, beams=beams)  # fxcorrelator.spead_issue_meta
```

Comment:
The spead receiver for the beamformer output products needs many of the correlator SPEAD metadata, as well as the beamformer specific info. If we can conditionally add these so that they are not sent if there is no beamformer then that would be fine.